docs/reference/mpremote.rst

```diff
@@ -198,49 +397,77 @@ executed: ``mount``, ``eval``, ``exec``, ``run``, ``fs``.  After doing a
 soft-reset for the first time, it will not be done again automatically, until a
 ``disconnect`` command is issued.
 
-Auto soft-reset behaviour can be controlled by the ``resume`` command.  And the
-``soft-reset`` command can be used to perform an explicit soft reset.
+Auto-soft-reset behaviour can be controlled by the ``resume`` command. This
+might be useful to use the ``eval`` command to inspect the state of of the
+device.  The ``soft-reset`` command can be used to perform an explicit soft
+reset in the middle of a sequence of commands.
 
+.. _mpremote_shortcuts:
 
 Shortcuts
 ---------
 
-Shortcuts can be defined using the macro system.  Built-in shortcuts are::
+Shortcuts can be defined using the macro system.  Built-in shortcuts are:
+
+- ``devs``: Alias for ``connect list``
```

Comment:
FWIW, I'm often asked how `devs` filters the serial device list to detect MicroPython devices.

Reply:
`devs` (i.e.`connect list`) doesn't filter, but `auto` does, so I have explained that.